src/control/cmd/agent/security_rpc.go

```diff
@@ -36,10 +40,10 @@ import (
 )
 
 // Module id for the Agent security module
-const securityModuleID int32 = 1
+const securityModuleID int32 = C.DRPC_MODULE_SEC_AGENT
```

Comment:
I'm on-board with using the C definitions, but it seems like it might be better to consolidate into a single .go file (types.go? modules.go?) so that you only need to add/maintain the cgo in one place.

Reply:
Ah, just noticed that this spans multiple packages. Still, maybe it makes sense to put these all in drpc/constants.go ?

Reply:
Yeah, I think that makes sense now that we're getting all the definitions from the same place anyway.